frontend/src/pages/projects/screens/detail/connections/ConnectionsDeleteModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { K8sStatus } from '@openshift/dynamic-plugin-sdk-utils';
+import { Connection } from '~/concepts/connectionTypes/types';
+import DeleteModal from '~/pages/projects/components/DeleteModal';
+
+type Props = {
+  deleteConnection: Connection;
+  onClose: () => void;
+  onDelete: () => Promise<K8sStatus>;
+  refresh: () => void;
```

Comment:
Instead of having a `refresh` prop, I prefer `onClose: (deleted: boolean) => void;` and have the consumer of the modal handle whatever it needs to on close.